shell/common/rasterizer.cc

```diff
@@ -752,7 +744,7 @@ DrawSurfaceStatus Rasterizer::DrawToSurfaceUnsafe(
     if (external_view_embedder_ &&
         (!raster_thread_merger_ || raster_thread_merger_->IsMerged())) {
       FML_DCHECK(!frame->IsSubmitted());
-      external_view_embedder_->SubmitFrame(
+      external_view_embedder_->SubmitView(
```

Comment:
I wonder if there's a clearer name for this that isn't overly long -- `SubmitViewFrame` for example?

Reply:
Oh, I thought this frame was like the "frame" as in "144 frames per second".